static/styles/settings.scss

```diff
@@ -280,6 +276,15 @@ td .button {
     border-top: none;
 }
 
+#stream-notify-settings .table {
+    border:none;
+}
+
+#stream-notify-settings .table thead tr th{
+    text-align: center;
+    vertical-align: middle;
+}
+
```

Comment:
What's the story with these changes?  They have some whitespace style issues, and in any case, there's no explanation as to where they come from in a commit whose description says it's just removing dead code.

Reply:
Oops, sorry! These were the changes in another issue I was working on, updated and removed them.